### PR TITLE
Change to use kyber.v1

### PR DESCRIPTION
**Pull request by jeffallen**
Also: simplify travis configs and de-flake a test.

### CODE REVIEW

File: service_test.go [hunk 1]
```diff
@@ -687,7 +687,7 @@ func waitOrFatal(ch chan bool, t *testing.T) {
 	select {
 	case _ = <-ch:
 		return
-	case <-time.After(500 * time.Millisecond):
+	case <-time.After(1500 * time.Millisecond):
```
Comment (kc1212) [inline service_test.go]: @jeffallen why this change? is it because travis is slow?

**Comment by ineiti** [comment]
I think I missed something here - I thought we're going for 

kyber.master <- onet.master <- cothority.master

and now onet.master should depend on kyber.v1?

**Comment by jeffallen** [comment]
I'm not certain what we want, this is an experiment to check that onet/cothority work right with kyber.v1. When we freeze onet and cothority (at v2, I think?) then they should be frozen to kyber.v1 as well, I guess. This was not ready for merge. that was an accident. I've moved it back to WIP.

File: service_test.go (hunk 1 above)
Comment (jeffallen) [reply service_test.go]: Maybe? It was an experiment. I'm not satisfied with it, but it is hard to see what else to do there.

**Comment by ineiti** [comment]
Thanks for the explanation - makes sense now!

**Comment by jeffallen** [comment]
This was an experiment. Closing for now.
